README.md

```diff
@@ -2,55 +2,35 @@
 
 # Project Name
```

Comment:
- [ ] As suggested by the previous reviewer, you are supposed to replace this `# Project Name` placeholder with your Project name. I can see that you placed it below but It will look more professional if you put it in this placeholder for your project name 😉